controllers/topics-controller.js

```diff
@@ -0,0 +1,9 @@
+const { allTopicsModel } = require("../models/topics-model");
+
+function getAllTopics(req, res) {
+  allTopicsModel().then((data) => {
+    res.status(200).send({ topics: data });
+  });
+}
+
```

Comment:
Some bits of feedback here:

- `getAllTopics` is a well name function name for this controller, well done.
- Response is well handled
- Sending the data back on a key of `topics` is appropriate, and a well named key.
- As you have a `.then` block, i'd advise adding a (and getting in the habit of) adding `.catch` block here to handle errors; even if it's just to `console.log(err)` at this stage before you implement your actual error handling middleware later.